### PR TITLE
sideframe

### PR DESCRIPTION
**Pull request by dhorelik**
- [x] Open sideframe through example.com > Admin
- [x] Reload and ensure sideframe is still visible
- [x] Close the sideframe again

users suite boilerplate setup done here as well

### CODE REVIEW

File: cms/tests/frontend/integration/settings/messages.js [hunk 1]
```diff
@@ -39,5 +44,7 @@ module.exports = {
 
             }
         }
+    },
+    users: {
```
Comment (ruslanmalogulko) [inline cms/tests/frontend/integration/settings/messages.js]: do we need empty property here?

Comment (dhorelik) [inline cms/tests/frontend/integration/settings/messages.js]: will fill this in the next PR =)

File: cms/tests/frontend/integration/logout.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ casper.test.begin('User Logout', function (test) {
             this.click('.cms-toolbar-item-navigation li:first-child a');
         })
         .waitForSelector('.cms-toolbar-item-navigation-hover', function () {
-            this.click('.cms-toolbar-item-navigation-hover ul li:last-child a');
+            this.click('.cms-toolbar-item-navigation-hover a[href$="/admin/logout/"]');
```
Comment (Brankub) [inline cms/tests/frontend/integration/logout.js]: :+1:

File: cms/tests/frontend/integration/sideframe.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+// #############################################################################
+// Admin panel opened in sideframe
+
+var globals = require('./settings/globals');
+var messages = require('./settings/messages').sideframe;
+
+casper.test.begin('Sideframe', function (test) {
+    casper
+        .start(globals.baseUrl, function () {
+            this.click('.cms-toolbar-item-navigation li:first-child a');
```
Comment (Brankub) [inline cms/tests/frontend/integration/sideframe.js]: could you rely on href?

**Comment by ruslanmalogulko** [comment]
:+1:

File: cms/tests/frontend/integration/sideframe.js (hunk 3 above)
Comment (dhorelik) [inline cms/tests/frontend/integration/sideframe.js]: nope, it's empty. This isn't robust, but there is no differentiation between nav items, really

Comment (Brankub) [inline cms/tests/frontend/integration/sideframe.js]: okay

**Comment by Brankub** [comment]
:+1:

**Pull request merged**